swf/src/types/drop_shadow_filter.rs

```diff
@@ -27,6 +27,11 @@ impl DropShadowFilter {
     pub fn num_passes(&self) -> u8 {
         (self.flags & DropShadowFilterFlags::PASSES).bits()
     }
+
+    #[inline]
+    pub fn hide_object(&self) -> bool {
```

Comment:
Per SWF19 CompositeSource should always be 1. Did you encounter real SWFs where it is 0?
By the way personally I prefer exposing a direct getter, i.e. `composite_source() -> bool` without a negation. The rational is to avoid abstraction where possible.

Reply:
> Per SWF19 CompositeSource should always be 1. Did you encounter real SWFs where it is 0? By the way personally I prefer exposing a direct getter, i.e. `composite_source() -> bool` without a negation. The rational is to avoid abstraction as possible.

This is the only time it's used, and it's "hide object" in the editor and ActionScript. Because it's "hide object", I figured we should stick to the inversion here and not cause the callers to always invert.